modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -0,0 +1,108 @@
+[[clients-slemicro]]
+= Registering {slemicro} Clients
+
+This section contains information about registering clients running these {slemicro} operating systems:
+
+* {slemicro}{nbsp}5.0 {x86_64}
+* {slemicro}{nbsp}5.0 {arm64}
+
+
+
+[WARNING]
+====
+This section is provided as a technology preview for testing purposes.
+Do not use this feature on production systems.
+====
+
+
+
+== Add Software Channels
+
+Before you register {slemicro} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
+The products you need for this procedure are:
+
+[[slemicro-channels-wizard]]
+[cols="1,1", options="header"]
+.SLE Micro Products - WebUI
+|===
+
+| OS Version
+| Product Name
+
+| {slemicro} 5.0 {x86_64}
+| SUSE Linux Enterprise Micro 5.0 aarch64
+
+| {slemicro} 5.0 {arm64}
+| SUSE Linux Enterprise Micro 5.0 x86_64
+
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[sle-channels-cli]]
+[cols="1,1,1", options="header"]
+.SLE Micro Products - CLI
+|===
+
+| OS Version
+| Base Channel
+| Updates Channel
+
+| {slemicro} 5.0 {x86_64}
+| suse-microos-5.0-pool-x86_64
+| suse-microos-5.0-updates-x86_64
+
+| {slemicro} 5.0 {arm64}
+| suse-microos-5.0-pool-aarch64
+| suse-microos-5.0-updates-aarch64
+
+|===
+
+include::snippets/addchannels_vendor_cli.adoc[]
+
+endif::[]
+
+
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{slemicro} channels can be very large.
```

Comment:
I seem to remember this is not the case.

openSUSE MicroOS -> small OS footprint, but you have the whole tumbleweed available
SLE Micro -> small OS footprint, and you have a limited set of packages.